docs/user_manual.md

```diff
@@ -332,6 +325,43 @@ These settings correspond to the ones in the [Docker docs][2].
 
 What is allowed here will vary based upon the discovery service plugin used.
 
+#### rolloutOptions
+When you run `helios rolling-update`, the rolling-update options will be constructed from the
+values you provide on the CLI and the job configuration with the following precedence:
+
+* Parameters included in the API request (eg, flags passed on the command line)
+* Parameters defined in the job configuration (of the job being rolled out)
+* The default value for the parameter as defined in the RolloutOptions class
+
+Example:
+
+`helios rolling-update --overlap --parallelism=3`
+
+with a job config containing
+
+```
+"rolloutOptions": {
+  "parallelism": 2,
+  "overlap": false,
+  "ignoreFailures": false
+}
+```
+
+Will ultimately have these options at `rolling-update` time. Since timeout, migrate, and token
```

Comment:
For clarity, I'd move the `Since timeout ...` part below the json snippet, otherwise "Will ultimately have these options at rolling-update time." looks like an incomplete sentence